lib/components/user/trip-basics-pane.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react'
+import { ButtonToolbar, ControlLabel, FormControl, FormGroup, ToggleButton, ToggleButtonGroup } from 'react-bootstrap'
+
+import { arrayToDayFields, dayFieldsToArray } from '../../util/monitored-trip'
+import TripSummary from './trip-summary'
+
+class TripBasicsPane extends Component {
```

Comment:
Please add some JSDoc for this component.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/174/commits/b4e99d7f6239a443ded667203754be15572ebf80